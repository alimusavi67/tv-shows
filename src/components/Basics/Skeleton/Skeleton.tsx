import React from 'react';

interface SkeletonProps {
  className?: string;
}

const Skeleton: React.FC<SkeletonProps> = ({ className }) => (
  <div data-testid="skeleton" className={`w-full h-40 bg-gray-300 animate-pulse ${className}`}></div>
);

export default Skeleton;
