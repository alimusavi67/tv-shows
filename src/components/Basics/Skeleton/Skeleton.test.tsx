import React from 'react';
import { render } from '@testing-library/react';
import Skeleton from './Skeleton';

test('renders Skeleton component correctly', () => {
  const { asFragment } = render(<Skeleton />);
  expect(asFragment()).toMatchSnapshot();
});
