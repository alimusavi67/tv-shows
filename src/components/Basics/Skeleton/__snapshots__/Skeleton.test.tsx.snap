// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Skeleton component correctly 1`] = `
<DocumentFragment>
  <div
    class="w-full h-40 bg-gray-300 animate-pulse undefined"
    data-testid="skeleton"
  />
</DocumentFragment>
`;
